src/cmdlib.sh

```diff
@@ -1102,3 +1102,22 @@ extract_osrelease_name() {
     # shellcheck disable=SC1091,SC2153
     (. "$out/os-release" && echo "${NAME}")
 }
+
+
+gen_ed25519_signing_key() {
+    local key_file="${1:-cosa_key}"
+    # Generate the key
+    openssl genpkey -algorithm ed25519 -outform PEM -out ${TMPDIR}/${key_file}
+
+    # Extract the pubkey
+    PUBKEY="$(openssl pkey -outform DER -pubout -in ${TMPDIR}/${key_file} | tail -c 32 | base64)"
+
+    ## write the pubkey in overrides
+    echo $PUBKEY > ${workdir}/overrides/rootfs/etc/ostree/initramfs-root-binding.key
```

Comment:
This should probably be 
 ```suggestion
    mkdir -p ${workdir}/overrides/initramfs/etc/ostree
    echo $PUBKEY > ${workdir}/overrides/initramfs/etc/ostree/initramfs-root-binding.key
```